ui/src/components/BeamlineActions/AnnotatedBeamlineActionDialog.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import { Row, Col, Modal, Button, Card } from 'react-bootstrap';
+import Plot1D from '../Plot1D';
+import { DraggableModal } from '../DraggableModal';
+import JSForm from '@rjsf/core';
+import validator from '@rjsf/validator-ajv8';
+import style from './BeamlineActions.module.css';
+
+export default function AnnotatedBeamlineActionDialog(props) {
+  const {
+    isDialogVisble,
+    handleOnHide,
+    defaultPosition,
+    actionName,
+    actionId,
+    actionSchema,
+    isActionRunning,
+    actionMessages,
+    handleStopAction,
+    handleStartAction,
+    handleOnPlotDisplay,
+    plotId,
+  } = props;
+
+  function getRunButton(isActionRunning) {
+    if (isActionRunning) {
+      return (
+        <Button
+          className={style.beamline_action_submit_button}
```

Comment:
`camelCase` class names by convention also. The class names can be shorter too, since they get namespaced.